kafka/overlays/smaug/alerts.yaml

```diff
@@ -0,0 +1,88 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: kafka-alerts
+spec:
+  groups:
+    - name: Kafka PVC
+      rules:
+        - alert: PVCFillingUp
+          annotations:
+            description: >-
+              The PersistentVolume claimed by {{ $labels.persistentvolumeclaim
+              }} in Namespace {{ $labels.namespace }} is only {{ $value |
+              humanizePercentage }} free.
+            summary: PersistentVolume is filling up.
+            # Tag the user who owns the PVC
```

Comment:
you can remove this comment since there is no user being tagged

Reply:
ok, just removed!